manifests/repo.pp

```diff
@@ -54,20 +54,21 @@
         repo_name => $repo_name,
       }
     }
-    'deb': {
-      $osname = downcase($::operatingsystem)
-      case $osname {
-        'debian', 'ubuntu': {
-
-          $component = 'main'
-          $repo_url = "${base_url}/${repo_name}/${osname}"
-          $distribution =  $::lsbdistcodename
-
-          file { $normalized_name:
```

Comment:
The `require => File[$normalized_name]` line should now also be removed from `Exec["apt_key_add_${normalized_name}"]` a bit further down, since `File[$normalized_name]` has now been replaced with an `apt::source` resource.